src/Castle.Windsor.Tests/ConventionVerification.cs

```diff
@@ -26,7 +26,8 @@ namespace CastleTests
 
 	using NUnit.Framework;
 
-	[Explicit]
+	[Ignore("Explicit Test, this fails on dotnet core")]
```

Comment:
This is ugly, I might raise an issue on NUnit for the adapter.

Reply:
Sounds like a defect in the NUnit runner.

I don't know why this test is failing and wasn't being run, but it appears we don't really need it anymore as we are nearly down to no obsolete members on those interfaces and classes, so we can just get rid of the whole test.